crates/edr_instrument/src/lib.rs

```diff
@@ -0,0 +1,2 @@
+/// Types for instrumenting code for the purpose of code coverage.
+pub mod coverage;
```

Comment:
Could potentially live in `edr_solidity`

Reply:
I moved this to a standalone crate as the idea is that this can be shipped in its own NPM package in the future. For the time being - as a shortcut - we're including it in one large NPM package with all other parts of EDR.

I considered including it in `edr_solidity` but it would require `slang` to be used as a dependency for that, even when we never would require it for compilation - in the future.